docs/quickstart.html

```diff
@@ -281,168 +281,9 @@
 
 <p>
 Kafka Streams is a client library of Kafka for real-time stream processing and analyzing data stored in Kafka brokers.
```

Comment:
I'd suggest to also change this line, notably because it somewhat suggests that not only the (original input) data is stored in Kafka brokers, but also that the Kafka Streams apps run inside the brokers.  (And yes, I do see we say "client library" at the beginning. :-P)

Perhaps:

> The Kafka Streams is a client library for building mission-critical real-time applications and microservices, where the input and/or output data is stored in Kafka clusters.  Kafka Streams combines the simplicity of writing and deploying standard Java and Scala applications with the benefits of Kafka's server-side cluster technology to make these applications highly scalable, elastic, fault-tolerant, distributed, and much more.